subsys/nrf_rpc/nrf_rpc_uart.c

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#include "zephyr/kernel.h"
```

Comment:
```suggestion
#include <zephyr/kernel.h>
```
super nit but other includes in this file follow a different style

Reply:
Moved to header file.